src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -949,6 +941,79 @@ public void testConsecutiveQueries() throws SQLException {
             }
         }
 
+        @Test
+        public void testMaxFetchSize() throws SQLException {
+            try (Connection con = getConnection(); Statement stmt = con.createStatement()) {
+
+                // Default value should return zero
+                long actual = stmt.getMaxFieldSize();
+                assertEquals(actual, (long) 0, "getMaxFieldSize():" + TestResource.getResource("R_incorrectDefault"));
```

Comment:
Expected value first. `assertEquals(long expected, long actual, String message)`

Reply:
No need to cast `0` to `long`.